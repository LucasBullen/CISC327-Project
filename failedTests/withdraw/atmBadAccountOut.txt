4c4
< Account doesn't exists. Withdraw cancelled.
---
> Account does not exist. Withdraw cancelled.
