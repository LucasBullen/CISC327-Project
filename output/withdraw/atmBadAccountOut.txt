Select mode: atm or agent.
Logged into atm mode.
Enter account number to withdraw from.
Account doesn't exists. Withdraw cancelled.
Logged out.
