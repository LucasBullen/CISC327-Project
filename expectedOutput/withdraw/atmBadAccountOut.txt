Select mode: atm or agent.
Logged into atm mode.
Enter account number to withdraw from.
Account does not exist. Withdraw cancelled.
Logged out.
